spec.md

```diff
@@ -56,4 +48,43 @@ The high-level components of the spec include:
 * Signatures that are based on signing image content address (optional OCI layer)
 * Naming that is federated based on DNS and can be delegated (optional OCI layer)
 
+## Actions
+
+The specified [components](#components) MAY be manipulated with actions.
+
+* **pack**: Create a blob from the local filesystem (or other storage).
+* **unpack**: Update the local filesystem (or other storage) based on data found in a blob.
```

Comment:
> `a blob` -> `series of tar-based blobs` ?

I'm using "(un)packing" more generically than that.  In the examples below, I also talk about (un)packing manifest(-list)s, and in that context it is "(de)serializing into JSON, potentially just in memory".